src/pages/fr/guides/deploy.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Déployer votre Site Astro
+description: Comment déployer votre site Astro sur le web.
+layout: ~/layouts/MainLayout.astro
+setup: import DeployGuidesNav from '~/components/DeployGuidesNav.astro';
+i18nReady: true
+---
+**Prêt à construire et déployer votre site Astro ?** Suivez un de nos guides sur différents services de déploiment, ou scrollez plus bas pour avoir des orientations générales sur comment déployer un site Astro.
+
+<DeployGuidesNav />
+
+## Options de déploiment rapide
+
+Vous pouvez construire et déployer rapidement un site Astro via de nombreux hébergeurs, via l'interface de leur site internet ou via une CLI.
+
+### Interface d'un hébergeur
+
+Une façon rapide de déployer votre site internet est de connecter le repository Git (ex. GitHub, GitLab, Bitbucket) de votre projet Astro à un hébergeur, afin de profiter du déploiment continu en utilisant Git.
```

Comment:
Translation and typo
```suggestion
Une façon rapide de déployer votre site internet est de connecter le dépôt Git (ex. GitHub, GitLab, Bitbucket) de votre projet Astro à un hébergeur, afin de profiter du déploiement continu en utilisant Git.
```

Reply:
Should we use "dépôt" instead of "repository" everywhere?

Reply:
Sorry, didn't see your replies pop in. Yeah I think so